docs/source/guides/time_series.ipynb

```diff
@@ -0,0 +1,284 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17f894b5",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')\n",
+    "import featuretools as ft\n",
+    "from featuretools.primitives import RollingMean, NumericLag, RollingMin\n",
+    "from featuretools.demo.weather import load_weather\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8104f18",
+   "metadata": {},
+   "source": [
+    "# Time Series Problems"
```

Comment:
How do you feel about calling this `Feature Engineering for Time Series Problems` since that is really the focus of this guide?